gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/service/DefaultGalleryService.java

```diff
@@ -49,19 +41,27 @@
 import org.silverpeas.components.gallery.model.Order;
 import org.silverpeas.components.gallery.model.OrderRow;
 import org.silverpeas.components.gallery.model.Photo;
-import org.silverpeas.components.gallery.process.GalleryProcessExecutionContext;
 import org.silverpeas.components.gallery.process.GalleryProcessManagement;
+import org.silverpeas.core.admin.component.model.ComponentInstLight;
 import org.silverpeas.core.admin.service.OrganizationController;
+import org.silverpeas.core.admin.space.SpaceInst;
+import org.silverpeas.core.admin.user.model.UserDetail;
+import org.silverpeas.core.date.period.Period;
+import org.silverpeas.core.exception.SilverpeasException;
+import org.silverpeas.core.exception.SilverpeasRuntimeException;
 import org.silverpeas.core.index.search.SearchEngineProvider;
 import org.silverpeas.core.index.search.model.MatchingIndexEntry;
 import org.silverpeas.core.index.search.model.QueryDescription;
-import org.silverpeas.core.date.period.Period;
+import org.silverpeas.core.node.dao.NodeDAO;
+import org.silverpeas.core.node.model.NodeDetail;
+import org.silverpeas.core.node.model.NodePK;
+import org.silverpeas.core.node.service.NodeService;
 import org.silverpeas.core.persistence.Transaction;
+import org.silverpeas.core.persistence.jdbc.DBUtil;
 import org.silverpeas.core.process.ProcessProvider;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.process.ProcessProvider'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)